\documentclass[10pt]{beamer}

% Beamer style
%\usetheme[secheader]{Madrid}
% \usetheme{CambridgeUS}
\useoutertheme{infolines}
\usecolortheme[rgb={0.65,0.15,0.25}]{structure}
% \usefonttheme[onlymath]{serif}
\beamertemplatenavigationsymbolsempty
%\AtBeginSubsection

% Packages
%\usepackage[french]{babel}
\usepackage[latin1]{inputenc}
\usepackage{color}
\usepackage{xspace}
\usepackage{dsfont, stmaryrd}
\usepackage{amsmath, amsfonts, amssymb}
\usepackage{epsfig}
\usepackage{tikz}
\usepackage{url}
\usepackage{/home/robin/LATEX/Biblio/astats}
%\usepackage[all]{xy}
\usepackage{graphicx}

% Commands
% \input{TikZcommands.tex}
\definecolor{darkred}{rgb}{0.65,0.15,0.25}
\newcommand{\emphase}[1]{\textcolor{darkred}{#1}}
% \newcommand{\emphase}[1]{{#1}}
\newcommand{\paragraph}[1]{\textcolor{darkred}{#1}}
\newcommand{\refer}[1]{{\small{\textcolor{blue}{{[\cite{#1}]}}}}}
% \newcommand{\Refer}[1]{{\small{\textcolor{gray}{{[#1]}}}}}
\renewcommand{\newblock}{}

% Symbols
\newcommand{\Abf}{{\bf A}}
\newcommand{\Beta}{\text{B}}
\newcommand{\Bcal}{\mathcal{B}}
\newcommand{\BIC}{\text{BIC}}
\newcommand{\Ccal}{\mathcal{C}}
\newcommand{\dd}{\text{~d}}
\newcommand{\dbf}{{\bf d}}
\newcommand{\Dcal}{\mathcal{D}}
\newcommand{\Esp}{\mathbb{E}}
\newcommand{\Ebf}{{\bf E}}
\newcommand{\Ecal}{\mathcal{E}}
\newcommand{\Gcal}{\mathcal{G}}
\newcommand{\Gam}{\mathcal{G}\text{am}}
\newcommand{\Hcal}{\mathcal{H}}
\newcommand{\Ibb}{\mathbb{I}}
\newcommand{\Ibf}{{\bf I}}
\newcommand{\ICL}{\text{ICL}}
\newcommand{\Cov}{\mathbb{C}\text{ov}}
\newcommand{\Corr}{\mathbb{C}\text{orr}}
\newcommand{\Var}{\mathbb{V}}
\newcommand{\Vsf}{\mathsf{V}}
\newcommand{\pen}{\text{pen}}
\newcommand{\Fcal}{\mathcal{F}}
\newcommand{\Hbf}{{\bf H}}
\newcommand{\Jcal}{\mathcal{J}}
\newcommand{\Kbf}{{\bf K}}
\newcommand{\Lcal}{\mathcal{L}}
\newcommand{\Mcal}{\mathcal{M}}
\newcommand{\mbf}{{\bf m}}
\newcommand{\mum}{\mu(\mbf)}
\newcommand{\mt}{\widetilde{m}}
\newcommand{\Nbb}{\mathbb{N}}
\newcommand{\Ncal}{\mathcal{N}}
\newcommand{\Nbf}{{\bf N}}
\newcommand{\Nm}{N(\mbf)}
\newcommand{\Ocal}{\mathcal{O}}
\newcommand{\Obf}{{\bf 0}}
\newcommand{\Omegas}{\underset{s}{\Omega}}
\newcommand{\Pbf}{{\bf P}}
\newcommand{\pt}{\widetilde{p}}
\newcommand{\Pt}{\widetilde{P}}
\newcommand{\Pcal}{\mathcal{P}}
\newcommand{\Qcal}{\mathcal{Q}}
\newcommand{\Rbb}{\mathbb{R}}
\newcommand{\Rcal}{\mathcal{R}}
\newcommand{\St}{\widetilde{S}}
\newcommand{\Scal}{\mathcal{S}}
\newcommand{\Tcal}{\mathcal{T}}
\newcommand{\Ucal}{\mathcal{U}}
\newcommand{\Vcal}{\mathcal{V}}
\newcommand{\BP}{\text{BP}}
\newcommand{\EM}{\text{EM}}
\newcommand{\VEM}{\text{VEM}}
\newcommand{\VBEM}{\text{VBEM}}
\newcommand{\cst}{\text{cst}}
\newcommand{\obs}{\text{obs}}
\newcommand{\ra}{\emphase{\mathversion{bold}{$\rightarrow$}~}}
%\newcommand{\transp}{\text{{\tiny $\top$}}}
\newcommand{\transp}{\text{{\tiny \mathversion{bold}{$\top$}}}}
\newcommand{\logit}{\text{logit}\xspace}

% Directory
\newcommand{\fignet}{/home/robin/Bureau/RECHERCHE/RESEAUX/EXPOSES/FIGURES}
\newcommand{\figchp}{/home/robin/Bureau/RECHERCHE/RUPTURES/EXPOSES/FIGURES}


%====================================================================
%====================================================================

%====================================================================
%====================================================================
\begin{document}
%====================================================================
%====================================================================

%====================================================================
\title[Probabilistic PCA for counts]{Probabilistic PCA for counts: A variational approach}

\author[S. Robin]{S. Robin \\ ~\\
  \begin{tabular}{ll}
    Joint work with J. Chiquet \& M. Mariadassou
  \end{tabular}
  }

\institute[INRA / AgroParisTech]{~ \\%INRA / AgroParisTech \\
  \vspace{-.1\textwidth}
  \begin{tabular}{ccc}
    \includegraphics[height=.25\textheight]{\fignet/LogoINRA-Couleur} & 
    \hspace{.02\textheight} &
    \includegraphics[height=.06\textheight]{\fignet/logagroptechsolo} % & 
%     \hspace{.02\textheight} &
%     \includegraphics[height=.09\textheight]{\fignet/logo-ssb}
    \\ 
  \end{tabular} \\
  \bigskip
  }

\date[May 2017, Rennes]{Hydrogen, May 2017, Rennes}

%====================================================================
%====================================================================
\maketitle
%====================================================================

%====================================================================
%====================================================================
\section{Multivariate analysis \& PCA}
\frame{\tableofcontents[currentsection]}
%====================================================================

%====================================================================
\subsection*{'Abundance' data}
%====================================================================
\frame{\frametitle{Abundance data}

  \paragraph{Data table.} 
  $$
  Y = [Y_{ij}]: n \times p, \qquad \text{either $n$ or $p$ 'large'}
  $$
  \begin{itemize}
   \item $Y_{ij} =$ count of $k$-mer $j$ in sample $i$
   \item $Y_{ij} =$ abundance of species $j$ in sample $i$
   \item ...
  \end{itemize}
  
  \bigskip \bigskip 
  \paragraph{Multivariate analysis:} summarize the information from $Y$.
}

%====================================================================
\subsection*{PCA}
%====================================================================
\frame{\frametitle{Principal component analysis (PCA)}

  \begin{tabular}{cc}
    \begin{tabular}{p{.4\textwidth}}
    Crime data:
    \begin{itemize}
     \item $n =51$ observations \\  
     \item $p = 4$ variables \\ 
     \item $q = 2$ principal components (PC)
    \end{itemize}
    \quad 
    \includegraphics[width=.35\textwidth]{../FIGURES/Fig-pPCA-exPCA-inertia.pdf}
    \end{tabular}
    & 
    \hspace{-.1\textwidth}
    \begin{tabular}{p{.5\textwidth}}
	 \includegraphics[width=.6\textwidth]{../FIGURES/Fig-pPCA-exPCA-biplot.pdf}
    \end{tabular}
  \end{tabular}
}

%====================================================================
\subsection*{pPCA}
%====================================================================
\frame{\frametitle{Probabilistic PCA (pPCA): V1}

  \paragraph{Gaussian model.} $Y_i = (Y_{i1}, \dots Y_{ip}) \sim \Ncal_p$

  \bigskip \bigskip 
  \paragraph{V1: Latent variable.} 
  $$
  W_i \text{ iid} \sim \Ncal_q(0, I_q), \qquad
  \underset{1 \times p}{Y_i} = \underset{1 \times p}\mu + \underset{1 \times q}{W_i} \;  \underset{q \times p}B + \underset{1 \times p}{E_i}
  $$
  where
  \begin{itemize}
   \item $\mu:$ mean vector
   \item $W_i:$ \emphase{principal coordinates} of observation $i$
   \item $B:$ matrix of coefficients
   \item $E_i:$ vector of non informative noise $\sim \Ncal_p(0, \sigma^2 I_p)$
  \end{itemize}
}

%====================================================================
\frame{\frametitle{Probabilistic PCA (pPCA): V2}

  \paragraph{Gaussian model.} $Y_i = (Y_{i1}, \dots Y_{ip}) \sim \Ncal_p$

  \bigskip \bigskip 
  \paragraph{V2: Marginal distribution.} 
  $$
  Y_i \text{ iid} \sim \Ncal_q(\mu, \Sigma), \qquad
  \underset{p \times p}{\Sigma} = B' B + \sigma^2 I_p
  $$
  where
  \begin{itemize}
   \item $\mu:$ mean vector
   \item $B:$ matrix of coefficients
   \item $\Sigma:$ \emphase{covariance matrix} 
  \end{itemize}
}

%====================================================================
\frame{\frametitle{Covariance structure}

  \begin{tabular}{cc}
    Empirical covariance & \hspace{-.15\textwidth} Rotated covariance \\
    \begin{tabular}{p{.5\textwidth}}
    \includegraphics[width=.45\textwidth]{../FIGURES/Fig-pPCA-Sigma.pdf}
    \end{tabular}
    & 
    \hspace{-.1\textwidth}
    \begin{tabular}{p{.5\textwidth}}
    \includegraphics[width=.45\textwidth]{../FIGURES/Fig-pPCA-rotSigma.pdf}
    \end{tabular} \\
    $p = 20$ & \hspace{-.15\textwidth} $q = 5$ 
  \end{tabular}
}

%====================================================================
\frame{\frametitle{Accounting for covariates}

  \paragraph{Additional information.}
  \begin{itemize}
   \item $x_i: 1 \times d$ vector of descriptors for observation $i$
   \item $\beta: d \times p$ matrix of regression coefficient (effect of each covariate on each species)
  \end{itemize}
  
  \bigskip \bigskip 
  \paragraph{Observation-specific mean.} 
  $$
  \mu \quad \rightarrow \quad \underset{1 \times p}{\mu_i} = \underset{1 \times d}{x_i} \; \underset{d \times p}{\beta}
  $$
}

%====================================================================
\frame{\frametitle{Inference}

  \paragraph{Incomplete data model:} 
  \begin{itemize}
   \item $Y :$ observed variables, 
   \item $W :$ unobserved variables, 
   \item $\theta = (B, \sigma^2):$ unknown parameters
  \end{itemize}

  \bigskip \bigskip 
  \paragraph{EM algorithm.} \refer{DLR77}
  \begin{itemize}
   \item E step: given $\theta$ compute $p_\theta(W_i|Y_i)$
   \item M step: update $\theta$ using the 'completed' likelihood $\Esp [\log p_\theta(Y, W) \;|\; Y]$
  \end{itemize}

  \bigskip \bigskip \pause
  \paragraph{Gaussian pPCA.} EM manageable because $p(W | Y)$ is explicit \refer{TiB99}. \\
  ... but we deal with non-Gaussian counts.
}

%====================================================================
%====================================================================
\section{pPCA for counts}
\frame{\tableofcontents[currentsection]}
%====================================================================

%====================================================================
\subsection*{Multivariate count data}
%====================================================================
\frame{\frametitle{Models for multivariate count data.}

  \paragraph{Abundance vector:} $Y_i = (Y_{i1}, \dots Y_{ip})$, $Y_{ij} \in \Nbb$

  \bigskip \bigskip \pause
  \paragraph{No generic model for multivariate counts.}
  \begin{itemize}
   \item Data transformation ($\widetilde{Y}_{ij} = \log(Y_{ij}), \sqrt(Y_{ij})$) \\ 
   \ra Pb when many counts are zero. \\ ~
   \item Poisson multivariate distributions \\
   \ra Constraints of the form of the dependency \refer{IYA16} \\ ~
   \item Latent variable models \\
   \ra Poisson-Gamma (negative binomial) \\
   \ra \emphase{Poisson-log normal (PLN)} \refer{AiH89}
  \end{itemize}

}

%====================================================================
\frame{\frametitle{{P}oisson-log normal (PLN) distribution}

  \paragraph{Latent Gaussian model:} 
  \begin{itemize}
   \item $Z_i:$ latent vector $\sim \Ncal_p(\mu, \Sigma)$ \\ 
   \item $(Y_{ij})_j:$ counts independent conditional on $Z_i$ \\
   \item $Y_{ij} \;|\; Z_{ij} \sim \Pcal\left(\lambda_{ij}\right), \qquad \log \lambda_{ij} = Z_{ij}$ \\ ~
  \end{itemize}

  \bigskip \pause
  \paragraph{Properties:}
  \begin{align*}
   \Esp(Y_{ij}) & = e^{\mu_j + \Sigma_{jj}/2} =: \lambda_j \\
   \Var(Y_{ij}) & = \lambda_j + \lambda_j^2 \left( e^{\Sigma_{jj}} - 1\right) 
   \qquad (\text{over-dispersion}) \\
   \Cov(Y_{ij}, Y_{ik}) & = \lambda_j \lambda_k \left( e^{\Sigma_{jk}} - 1\right)
   \quad \qquad (\text{arbitrary sign}) 
  \end{align*}
}

%====================================================================
\subsection*{PLN-PCA}
%====================================================================
\frame{\frametitle{pPCA for count data}

  \paragraph{Idea.} Merge pPCA with Poisson log-normal \refer{CMR17}.
  
  \bigskip \bigskip \pause
  \begin{tabular}{p{.5\textwidth}p{.5\textwidth}}
  \paragraph{Latent variable:}
  \begin{itemize}
  \item $W_i \sim \Ncal_q(0, I_q)$ 
  \item $Z_i = \mu + W_i B$
  \end{itemize}
  &
  \paragraph{Covariance structure:}
  \begin{itemize}
  \item $Z_i \sim \Ncal_q(\mu, \Sigma)$ 
  \item $\Sigma = B' B$
  \end{itemize}
  \end{tabular}
  \begin{itemize}
   \item $(Y_{ij})$ independent $| \; (Z_{ij}):$ $Y_{ij} \sim \Pcal\left(\lambda_{ij}\right), \qquad \log \lambda_{ij} = Z_{ij}$
  \end{itemize}
  
  \bigskip \bigskip \pause
  \paragraph{Covariates:} $Z_i = x_i \beta + W_i B$ \quad or \quad $Z_i \sim \Ncal_q(x_i \beta, \Sigma)$ 
}

%====================================================================
\frame{\frametitle{Inference}

  \paragraph{EM algorithm does not apply} because $p(W_i \; | \; Y_i)$ intractable.
  
  \bigskip \bigskip \pause
  \paragraph{Variational approximation.}
  $$
  p(W_i \; | \; Y_i) \approx \pt(W_i) = \Ncal_q(\mt_i, \St_i)
  $$
  $\mt_i$, $\St_i$: variational parameters.

  \bigskip \bigskip \pause
  \paragraph{Variational EM.} Aim at maximizing a lower bound
  $$
  \Jcal(Y; \theta, \pt) \leq \log p_\theta(Y)
  $$
  \paragraph{Property.} $\Jcal(Y; \theta, \pt)$ is convex wrt $(B, \mu)$ and wrt $(\mt, \St)$ \\
  \ra No need of E and M steps.
  
}

%====================================================================
\frame{\frametitle{Model selection}

  \paragraph{Number of component $q$:} needs to be chosen.
  
  \bigskip \bigskip \pause
  \paragraph{Penalized 'likelihood'.}
  \begin{itemize}
   \item $\log p_{\widehat{\theta}}(Y)$ intractable: replaced with $\Jcal(Y; {\widehat{\theta}}, \pt)$ \\ ~
   \item $vBIC_q = \Jcal(Y; {\widehat{\theta}}, \pt) - pq \log(n)/2$ \refer{Sch78} \\ ~ 
   \item $vICL_q = vBIC_q - \Hcal(\pt)$ \refer{BCG00} \\ ~ 
  \end{itemize}
  
  \bigskip
  \paragraph{Chosen rank:}
  $$
  \widehat{q} = \arg\max_q vBIC_q
  \qquad \text{or} \qquad
  \widehat{q} = \arg\max_q vICL_q
  $$

}

%====================================================================
%====================================================================
\section{Illustrations}
\frame{\tableofcontents[currentsection]}
%====================================================================

%====================================================================
\frame{\frametitle{Metagenomic experiment}

  \paragraph{Data.} Metabarcoding
  $$
  Y_{ij} = \text{number of reads associated with species $j$ in sample $i$}
  $$

  \bigskip \bigskip \pause
  \paragraph{Generic model.} 
  $$
  Y_{ij} \sim \Pcal(\lambda_{ij}), \qquad \log \lambda_{ij} = o_i + x_j \beta_j + W_i B_j
  $$
  \begin{itemize}
   \item $o_i =$ offsets: sequencing depth in sample $i$ for bacteria (fungi)
   \item $x_i =$ covariates
   \item $W_i =$principal coordinates
  \end{itemize} 
}

%====================================================================
\subsection*{Pathobiome}
%====================================================================
\frame{\frametitle{Pathobiome: Oak powdery mildew}

  \paragraph{Data from \refer{JFS16}.} 
  \begin{itemize}
   \item $n = 116$ oak leaves = samples
   \item $p_1 = 66$ bacterial species (OTU)
   \item $p_2 = 48$ fungal species ($p = 114$)
   \item covariates: tree (resistant, intermediate, susceptible), height, distance to trunk, ...
   \item offsets: $o_{i1}, o_{i2} =$ offset for bacteria , fungi
  \end{itemize}

  \bigskip \bigskip \pause
  \paragraph{Aim.} Understand the interaction between the species, including the oak mildew pathogene {\sl E. alphitoides}.
}

%====================================================================
\frame{\frametitle{Pathobiome: How many components?}

  $$
  \includegraphics[width=.9\textwidth]{../FIGURES/CMR17-Fig1a_ModSel.pdf}
  $$
  $M_0$ (left): offset only; $M_1$ (right): offset + covariates, 
}

%====================================================================
\frame{\frametitle{Pathobiome: First 2 PCs}

  $$
  \includegraphics[width=.9\textwidth]{../FIGURES/CMR17-Fig1c_IndMap.pdf}
  $$
  $M_0$ (left): offset only; $M_1$ (right): offset + covariates, 
}


%====================================================================
\frame{\frametitle{Pathobiome: Precision of $\widehat{Z}_{ij}$}
  
  $$
  \includegraphics[width=.5\textwidth]{../FIGURES/CMR17-Fig2_VarZ.pdf}
  $$
  $x-$axis: $Y_{ij}$; $y-$axis: approximate sd of $Z_{ij}$
  
  \bigskip
  Due to the link function (log) $\Var(Z_{ij})$ is higher when $Y_{ij}$ is close to 0.
}

%====================================================================
\subsection*{Microbiome}
%====================================================================
\frame{\frametitle{Microbiome: Impact of weaning on piglet}

  \paragraph{Data from \refer{MBE15}.} 
  \begin{itemize}
   \item $n = 115$ samples (= $51$ piglets at $5$ times)
   \item $p = 4031$ bacterial species (OTU)
%    \item covariates: piglet, time
   \item offsets: $o_i =$ offset for sample $i$
  \end{itemize}

  \bigskip \bigskip \pause
  \paragraph{Aim (here).} Impact of considering only a fraction of (most abundant) species.
}

%====================================================================
\frame{\frametitle{Microbiome: Computation time}

  $$
  \includegraphics[width=.5\textwidth]{../FIGURES/CMR17-Fig3_timings.pdf}
  $$
  $x-$axis: number of species; $y-$axis: computation time (s)
}

%====================================================================
\frame{\frametitle{Microbiome: Computation time}

  \begin{tabular}{ccc}
    $\log_{10}$ abundances & $R^2$ criterion & chosen rank $\widehat{q}$ \\
    \includegraphics[width=.3\textwidth]{../FIGURES/CMR17-Fig4a-CritAbundance.pdf}
    &
    \includegraphics[width=.3\textwidth]{../FIGURES/CMR17-Fig4b-CritR2.pdf}
    &
    \includegraphics[width=.3\textwidth]{../FIGURES/CMR17-Fig4c-CritRank.pdf}
  \end{tabular}
  $x-$axis: number of species
}

%====================================================================
\subsection*{Hydrogen}
%====================================================================
\frame{\frametitle{Hydrogen: 21 Tara samples}

  \paragraph{Same old dataset.} 
  \begin{itemize}
   \item $n = 21$ samples 
   \item $p = 4^{10}$ 10-mers frequencies 
   \item reduced to $p = 10000$ after chi-square test filtering
   \item offset = log-cumulated count for selected $k$-mers
   \item no covariates
   \item removing a 'main effect' manually
  \end{itemize}

  \bigskip \bigskip \pause
  \paragraph{Aim (here).} 
  \begin{itemize}
   \item Preliminary analyses
   \item Compare with PCA on log-transformed counts $\log(1+Y_{ij}) - \log(1+Y_{+j})$
  \end{itemize}
}

%====================================================================
\frame{\frametitle{Hydrogen: Model selection}

  \begin{tabular}{cc}
   log-PCA & \hspace{-.1\textwidth} PLN-PCA \\
   \begin{tabular}{c}
   \includegraphics[width=.5\textwidth]{../FIGURES/Hydrogene_k10_correctTRUE_logPCA_ModSel.pdf}
   \end{tabular}
   & \hspace{-.02\textwidth}
   \begin{tabular}{c}
   \includegraphics[width=.4\textwidth]{../FIGURES/Hydrogene_k10_correctTRUE_PLNPCA_ModSel.pdf}
   \end{tabular}
  \end{tabular}
  
  \ra Take $\widehat{q} = 7$ (max $R^2$) for PLN-PCA
}

%====================================================================
\frame{\frametitle{Hydrogen: Sample map (1-2)}

  \begin{tabular}{cc}
   log-PCA & \hspace{-.1\textwidth} PLN-PCA \\
   \begin{tabular}{c}
   \includegraphics[width=.5\textwidth]{../FIGURES/Hydrogene_k10_correctTRUE_logPCA_Ind1-2.pdf}
   \end{tabular}
   & \hspace{-.02\textwidth}
   \begin{tabular}{c}
   \includegraphics[width=.42\textwidth]{../FIGURES/Hydrogene_k10_correctTRUE_PLNPCA_K7_Ind1-2.pdf}
   \end{tabular}
  \end{tabular}
}

%====================================================================
\frame{\frametitle{Hydrogen: $k$-mer circle (1-2)}

  \begin{tabular}{cc}
   log-PCA & \hspace{-.1\textwidth} PLN-PCA \\
   \begin{tabular}{c}
   \includegraphics[width=.5\textwidth]{../FIGURES/Hydrogene_k10_correctTRUE_logPCA_Var1-2.pdf}
   \end{tabular}
   & \hspace{-.02\textwidth}
   \begin{tabular}{c}
   \includegraphics[width=.42\textwidth]{../FIGURES/Hydrogene_k10_correctTRUE_PLNPCA_K7_Var1-2.pdf}
   \end{tabular}
  \end{tabular}
}

%====================================================================
\frame{\frametitle{Hydrogen: PC 3-4}

  \vspace{-.05\textheight}
  $$
  \begin{tabular}{cc}
%    log-PCA & \hspace{-.1\textwidth} PLN-PCA \\
   \begin{tabular}{c}
   \includegraphics[width=.3\textwidth]{../FIGURES/Hydrogene_k10_correctTRUE_logPCA_Ind3-4.pdf}
   \end{tabular}
   & \hspace{-.02\textwidth}
   \begin{tabular}{c}
   \includegraphics[width=.24\textwidth]{../FIGURES/Hydrogene_k10_correctTRUE_PLNPCA_K7_Ind3-4.pdf}
   \end{tabular} \\
   \begin{tabular}{c}
   \includegraphics[width=.3\textwidth]{../FIGURES/Hydrogene_k10_correctTRUE_logPCA_Var3-4.pdf}
   \end{tabular}
   & \hspace{-.02\textwidth}
   \begin{tabular}{c}
   \includegraphics[width=.24\textwidth]{../FIGURES/Hydrogene_k10_correctTRUE_PLNPCA_K7_Var3-4.pdf}
   \end{tabular}
  \end{tabular}
  $$
}

%====================================================================
%====================================================================
\section{Discussion}
\frame{\tableofcontents[currentsection]}
%====================================================================

%====================================================================
\frame{\frametitle{Summary}

  \begin{itemize}
   \item Generic PLN model combining pPCA and GLM
   \item Efficient variational algorithm
   \item Model selection criteria
  \end{itemize}
  
  \bigskip \pause
  \paragraph{And...}
  \begin{itemize}
   \item Visualization (non-nested latent spaces when $q$ increases)
   \item Goodness-of-fit criteria
   \item Package available at \emphase{\tt https://github.com/jchiquet/PLNmodels} \\
   \ra Try it!
  \end{itemize}
  $$
  \text{\url{
  PCA = PLNPCA(Y ~ 1 + X + offset(O), Q=1:10, control = ...)
  }}
  $$

}

%====================================================================
\frame{\frametitle{Extensions}

  \paragraph{For Hydrogen}
  \begin{itemize}
   \item Use it to analyze $k$-mers across larger Tara samples datasets 
   \item Use it to account for environmental covariates
  \end{itemize}
  
  \bigskip \bigskip \pause
  \paragraph{More generally}
  \begin{itemize}
   \item Force sparsity in $B$
   \item Extension to other distributions (Bernoulli for 0/1 data, ...) 
   \item Alternative modeling of $\Sigma$ for network inference
  \end{itemize}

}

%====================================================================
\frame{ \frametitle{References}
{\tiny
  \bibliography{/home/robin/Biblio/BibGene}
%   \bibliographystyle{/home/robin/LATEX/Biblio/astats}
  \bibliographystyle{alpha}
  }
}

%====================================================================
%====================================================================
% \appendix 
\section{Appendix}
%====================================================================

%====================================================================
\subsection*{Goodness of fit}
%====================================================================
\frame{\frametitle{Appendix: Goodness of fit}

  \paragraph{pPCA:} Cumulated sum of the eigenvalues = \% of variance preserved on the first $q$ components.
  
  \bigskip \bigskip 
  \paragraph{PLN-pPCA:} Deviance based criterion.
  \begin{itemize}
   \item Compute $\widetilde{Z}^q = O + X \widehat{\beta} + \widetilde{M}^q \widehat{B}^q$
   \item Take $\lambda_{ij}^q = \exp(\widetilde{Z}_{ij}^q)$
   \item Define $\lambda_{ij}^{\min} = \exp( \widetilde{Z}_{ij}^0)$ and $\lambda_{ij}^{\max} = Y_{ij}$ 
   \item Compute the Poisson log-likelihood $\ell_q = \log P(Y; \lambda^q)$
  \end{itemize}
  
  \bigskip
  \paragraph{Pseudo-$R^2$:} 
  $$
  R_q^2 = \frac{\ell_q - \ell_{\min}}{\ell_{\max} - \ell_{\min}}
  $$
}

%====================================================================
\frame{\frametitle{Pathobiome: Goodness of fit}
  
  $$
  \includegraphics[width=.9\textwidth]{../FIGURES/CMR17-Fig1b_GoodnessEntropy.pdf}
  $$
  $x-$axis: $q$; $y-$axis: left = $R_q^2$, right = $-\Hcal(\pt) = vBIC_q vICL_q$
}

%====================================================================
\subsection*{Visualization}
%====================================================================
\frame{\frametitle{Appendix: Visualization}

  \paragraph{PCA:} Optimal subspaces nested when $q$ increases.
  
  \bigskip \bigskip 
  \paragraph{PLN-pPCA:} Non-nested subspaces.
  \begin{itemize}
   \item For a given dimension $q$
   \item Compute the estimated latent positions $\widetilde{P} = \widetilde{M} \widehat{B}$
   \item Perform PCA on the $\widetilde{P}$.
  \end{itemize}
}

%====================================================================
\subsection*{PCA as usual}
%====================================================================
\frame{\frametitle{Appendix: PCA as usual (Piglets)}

  $$
  \includegraphics[width=.55\textwidth]{../FIGURES/CMR17-Fig5-PCA-axes-12.pdf}
  $$
  \ra Or use $\widehat{B}$
}

%====================================================================
%====================================================================
\end{document}
%====================================================================
%====================================================================

  \begin{tabular}{cc}
    \begin{tabular}{p{.5\textwidth}}
    \end{tabular}
    & 
    \hspace{-.02\textwidth}
    \begin{tabular}{p{.5\textwidth}}
    \end{tabular}
  \end{tabular}

